akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorTest.java

```diff
@@ -26,7 +26,7 @@
 public class PersistentActorTest {
 
   @ClassRule
-  public static final TestKitJunitResource testKit = new TestKitJunitResource(PersistentActorSpec$.MODULE$.config());
+  public static final TestKitJunitResource testKit = new TestKitJunitResource(PersistentBehaviorSpec$.MODULE$.config());
```

Comment:
boy scout: add some accessor instead of using MODULE

Reply:
Running into 

```
error] /Users/ktoso/code/akka/akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorTest.java:29:1: non-static method config() cannot be referenced from a static context
[error]   public static final TestKitJunitResource testKit = new TestKitJunitResource(PersistentBehaviorSpec.config());
```

Reply:
Made a ticket https://github.com/akka/akka/issues/24692